trajectory_msgs/JointTrajectory trajectory
---
float64 length
#float64 smoothness
#float64 clearance
